phases/17-libxslt.bat

```diff
@@ -27,6 +27,7 @@ cscript configure.js ^
   %CONFIGURE_OPTS% ^
   "prefix=%INSTALL_PREFIX%" ^
   "include=%INSTALL_PREFIX%\include" ^
+  "include=%INSTALL_PREFIX%\include\libxml2" ^
```

Comment:
Why is this needed? Does the libxml2 include path change with this PR?

Reply:
libxml2 CMake installs libxml/*.h into PREFIX/include/libxml2 instead of just PREFIX/include. The old libxslt configure.js does not seem to use pkgconfig to retrieve the CFLAGs from libxml2.

Reply:
I see. I guess ideally we’d also update libxslt to CMake, but this seems fine for now. 👍